packages/lib/src/shared_commands.ts

```diff
@@ -123,16 +126,29 @@ export function configCommand(yargs: any) {
  * Handle the actions that are made available by configCommand.
  *
  * @param args - yargs args object.
- * @param instance - Config instance.
+ * @param instanceLoader - Function that loads the config instance.
  * @param configPath - Path to configuration file.
+ * @param emptyConfigSupplier - Function supplying a default config for
+ *   the create command.
  */
 export async function handleConfigCommand(
 	args: Record<string, unknown>,
-	instance: libConfig.Config<any>,
-	configPath: string
+	instanceLoader: () => Promise<libConfig.Config<any>>,
+	configPath: string,
+	emptyConfigSupplier: () => libConfig.Config<any>
 ) {
 	let command = (args._ as string[])[1];
 
+	if (command === "create") {
+		if (await fs.exists(configPath)) {
+			logger.error(`Config "${configPath}" already exists`);
+			return;
+		}
+		await libFileOps.safeOutputFile(configPath, JSON.stringify(emptyConfigSupplier(), null, "\t"));
```

Comment:
Rather than using `fs.exists` you should use a try catch for `EEXIST`. Also as the happy path here is that the file does not exist, and because the config file is quite small, it is better to use `fs.outputFile` rather than `safeOutputFile` since the safe version is designed to overwrite existing files that can't risk being corrupted.